IceNet/polar-modelling-icenet.ipynb

```diff
@@ -0,0 +1,1411 @@
+{
```

Comment:
<p>This path will depend on where the notebook is being run - I guess it makes sense to leave the one here as the one that works on the AzureML VM, but maybe could add a note to the text to point that out?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/PangeoExamples/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I'll add a the following text:</p><p> </p><blockquote>Note the python route after the exclamation in the line below works on the AzureML VM. The route might change in your local system. </blockquote><p> </p><p>How does it sound?</p>